hack/e2e.sh

```diff
@@ -15,15 +15,16 @@ if [[ -z "$SUITE" || "$SUITE" == "enduser" ]]; then
   # Login as osadmin to simulate a regular user
   password=$(awk '/^  adminPasswd:/ { print $2 }' <_data/containerservice.yaml)
   fqdn=$(awk '/^  fqdn:/ { print $2 }' <_data/containerservice.yaml)
-  export KUBECONFIG=_data/_out/osadmin.kubeconfig
```

Comment:
This change is going to break the test because `oc login` will not generate the kubeconfig in this path.

Reply:
Maybe make the path absolute by using `pwd`.